.github/pull_request_template.md

```diff
@@ -1,5 +1,8 @@
 **What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**
 
-**Give a brief summary of what the PR does, explaining any non-trivial design decisions**
+---
 
+**Give a succinct summary of what the PR does, explaining any non-trivial design decisions**
```

Comment:
Just remove "succinct". Should be:
Give a summary of what the PR does

Reply:
```suggestion
**Give a summary of what the PR does, explaining any non-trivial design decisions**
```